h2o-py/tests/testdir_apis/Data_Manipulation/pyunit_h2oH2OFrame_day.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import print_function
+import sys
+sys.path.insert(1,"../../../")
+from tests import pyunit_utils
+import h2o
+from h2o.utils.typechecks import assert_is_type
+from h2o.frame import H2OFrame
+
+def h2o_H2OFrame_day():
```

Comment:
see if all .day, .month, .year, .week, .dayOfWeek, .hour, .minute, .sec, .msec can be combined into a single test, checking against python (or pandas) standard datetime functionality

Reply:
Done